L1Trigger/TrackTrigger/interface/Setup.h

```diff
@@ -154,6 +160,18 @@ namespace tt {
     double maxdPhi() const { return maxdPhi_; }
     // maximum representable stub z uncertainty
     double maxdZ() const { return maxdZ_; }
+    // barrel layer limit z value to partition into tilted and untilted region
+    double tiltedLayerLimitZ(int layer) const { return tiltedLayerLimitsZ_.at(layer); }
+    // endcap disk limit r value to partition into PS and 2S region
+    double psDiskLimitR(int layer) const { return psDiskLimitsR_.at(layer); }
+    // strip pitch of outer tracker sensors in cm
+    double pitch2S() const { return pitch2S_; }
```

Comment:
aren't these things that should be read from the geometry directly?

Reply:
see comment below